File: 2000 98 & ME/QuickStart/howto/samples/IO/readwrite/readwrite.cs
using System;
using System.IO;

class ReadWrite
{
    
    public static void Main(String[] args)
    {
        FileStream fs = new FileStream("data.txt", FileMode.OpenOrCreate);	
        BinaryWriter w = new BinaryWriter(fs);
        BinaryReader r = new BinaryReader(fs);
        for (int i = 0; i < 11; i++)
        {
            w.Write( (int) i);
        }
        
        w.BaseStream.Seek(0,SeekOrigin.Begin);	//set the file pointer to the beginning
        for (int i = 0; i < 11; i++)
        {
            Console.WriteLine( r.ReadInt32() );
        }
        fs.Close();

		Console.WriteLine ("\r\nPress Return to exit.");
		Console.Read();
    }
    
}

File: 2000 98 & ME/QuickStart/howto/samples/IO/readwrite/VB/readwrite.vb

Imports System.IO
Imports System

Public Class ReadWrite
    
    Public Shared Sub Main()
    
        Dim fs As FileStream 
        fs = new FileStream("data.txt", FileMode.OpenOrCreate)	
        Dim w As BinaryWriter 
        w = new BinaryWriter(fs)
        Dim r As BinaryReader 
        r = new BinaryReader(fs)
        Dim I as Integer
        for I = 0 to 11 
            w.Write( CInt ( i))
        Next I
        
        w.BaseStream.Seek(0,SeekOrigin.Begin)	'set the file pointer to the beginning
        for I = 0 to 11
            Console.WriteLine( r.ReadInt32() )
        Next I
        fs.Close()

		Console.WriteLine ()
		Console.WriteLine ("Press Return to exit.")
		Console.Read()
    End Sub
    
End Class
